subworkflows/local/visualization_krona.nf

```diff
@@ -0,0 +1,57 @@
+//
+// Create Krona visualizations
+//
+
+include { KRAKENTOOLS_KREPORT2KRONA } from '../../modules/nf-core/modules/krakentools/kreport2krona/main'
+include { KRONA_CLEANUP             } from '../../modules/local/krona_cleanup'
+include { KRONA_KTIMPORTTEXT        } from '../../modules/nf-core/modules/krona/ktimporttext/main'
+
+workflow VISUALIZATION_KRONA {
+    take:
+    profiles
+
+    main:
+    ch_krona_text = Channel.empty()
+    ch_krona_html = Channel.empty()
+    ch_versions = Channel.empty()
+
+    /*
+        Split profile results based on tool they come from
+    */
+    ch_input_profiles = profiles
+        .branch {
+            centrifuge: it[0]['tool'] == 'centrifuge'
+            kraken2: it[0]['tool'] == 'kraken2'
+            unknown: true
+        }
+
+    /*
+        Convert Kraken2 formatted reports into Krona text files
```

Comment:
```suggestion
        Convert Kraken2 and Centrifuge formatted reports into Krona text files
```

Reply:
Technically, these aren't reports in Centrifuge format (they've been converted by `CENTRIFUGE_KREPORT`). I can add a note of that, if you'd like.

Reply:
Yes please! That would be helpful.You can do this in a separate PR though as the tests are passing  now (and take a long time)